requirements/testing.txt

```diff
@@ -107,8 +125,6 @@ pydata-google-auth==1.7.0
     # via pandas-gbq
 pyfakefs==5.2.2
     # via -r requirements/testing.in
-pyhive[presto]==0.6.5
```

Comment:
This is likely a legacy dependency resulting from not having run `pip-compile-multi`. The `presto` extra requirement is specified in `requirements/development.in` which is an input to `requirements/testing.in` and thus shouldn't be re-included here.